### DISCUSSION TITLE
Coding Style Guide

### DISCUSSION THREAD

**Discussion by victorray84** [inovex/scrumlr.io#4146] [Q&A] [answered]
I want to contribute to the backend part. is there any document for coding style guide??

**Comment by brandstetterm** [comment]
Hey @victorray84, 
Thanks for reaching out! It's great to hear that you're considering contributing to Scrumlr! We currently do not have specific coding style guidelines for the backend. Other important information like the definition of done or our pull request guidelines can be found in our [Contribution Guide](https://github.com/inovex/scrumlr.io/blob/main/CONTRIBUTING.md).

**Comment by victorray84** [comment]
Hey @brandstetterm ,
      I'm going through ur backend code.  Do u have any future backend roadmap for any new feature or enhancement??

**Reply by Schwehn42** [reply, marked as answer]
hi, you can find the backlog here: https://github.com/orgs/inovex/projects/4/views/1?filterQuery=team%3A%22Dev+-+Backend%22

most tasks aren't refined yet, but I hope it helps anyway!

**Reply by victorray84** [reply]
Thanks for sharing . I'd look into this issue.
